parastell/invessel_build.py

```diff
@@ -622,24 +646,24 @@ def _tag_volumes_with_materials_pydagmc(self):
 
     def generate_components_pydagmc(self):
         """Use PyDAGMC to build a DAGMC model of the invessel components"""
-        if np.isclose(
-            (self._repeat + 1) * self.radial_build.toroidal_angles[-1], 360
-        ):
-            e = AssertionError(
-                "The PyDAGMC workflow does not support modeling 360-degree "
-                "geometries. For configurations with more than one period, "
-                "please consider modeling only one period. i.e. set "
-                "'repeat = 0'."
-            )
-            self._logger.error(e.args[0])
-            raise e
         self._logger.info(
             "Generating DAGMC model of in-vessel components with PyDAGMC..."
         )
+
+        if (
+            self.radial_build.toroidal_angles[-1]
+            - self.radial_build.toroidal_angles[0]
+            == 360.0
```

Comment:
Do we want to test for this to be some version of close to 360 rather than exactly 360? Like we originally did...

Reply:
Yes, that's my bad